package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "menubar",
-  "version": "4.1.0",
+  "version": "4.2.0",
   "description": "high level way to create menubar desktop applications with electron",
   "main": "index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/maxogden/menubar.git"
   },
   "scripts": {
+    "start": "electron .",
```

Comment:
It's useful when user has `electron-prebuilt` or some electron bin package as the dependencies. I think it's better to omit it. 🤔